arc.pl

```diff
@@ -809,32 +808,33 @@ sub make_ephin_goes_table {
     my %val;
     my %tab_def = %{$opt{ephin_goes_table}};
 
-    my $start = qr/P1 \s+ P2  \s+ P5 \s+ P8  \s+ P10 \s+ P11 \s+ H2/x;
-
-    my ($goes_date, $p2, $p5);
-    if (defined $web_data->{goes}{content}{flux}{content}){
-        ($goes_date, $p2, $p5) = parse_mta_rad_data($start,
-                                                    $web_data->{goes}{content}{flux}{content},
-                                                    5, 6,
-                                                );
-    }
-    $goes_date = 'UNAVAILABLE' unless defined $goes_date;
-
     my ($hrc_shield_proxy, $hrc_time) = split(' ', io($opt{file}{hrc_shield})->slurp());
     my ($p4gm_proxy, $p4gm_time) = split(' ', io($opt{file}{p4gm})->slurp());
     my ($p41gm_proxy, $p41gm_time) = split(' ', io($opt{file}{p41gm})->slurp());
 
-    my $warning = ((not defined $p2) || (not defined $p5) || @{$p2} == 0 || @{$p5} == 0) ?
-      '<h2 style="color:red;text-align:center">NO RECENT GOES DATA<br/> NO data for HRC PROXY</h2>' : '';
-
     my $ephin_date = $snap->{obt}{value} . ' (' .
 		  Event::calc_delta_date($snap->{obt}{value}) . ')';
     my $hrc_delta = Event::calc_delta_date($hrc_time);
     my $p4gm_delta = Event::calc_delta_date($p4gm_time);
     my $p41gm_delta = Event::calc_delta_date($p41gm_time);
 
-    $val{GOES}{P4GM}  = (defined $p2 and @{$p2}) ? sprintf("%.2f", $p4gm_proxy) : '---'; # See http://asc.harvard.edu/mta/G10.html
-    $val{GOES}{P41GM} = (defined $p5 and @{$p5}) ? sprintf("%.2f", $p41gm_proxy) : '---'; # ditto
+    # Add a warning above the CXO GOES rates table if the proton data for these
+    # is stale.  Since these times are all from the proton/get_hrc data they
+    # should be the same, but that implementation could change.
+    my $warning = '<h2 style="color:red;text-align:center">';
+    if (($CurrentTime - $hrc_time) > 86400){
```

Comment:
We were already printing the "delta" time for each of these metrics underneath the table, so these are just red warnings above the table to call attention to stale data.  I figured that would be more obvious than updating the font color in the fine print under/within the CXO GOES rates table.  I tested this by just setting the 86400 to a really small value and confirming expected deltas and warning.

Reply:
@taldcroft if you agree this approach is somewhat acceptable, I think we should be done with this PR for now (we need future feedback on P41GM and other limits but probably doesn't need to be in here).